BucStop/Views/Shared/_Layout.cshtml

```diff
@@ -55,7 +56,7 @@
     <!-- This block handles the footer, which includes the version number for the project. -->
     <footer class="border-top footer text-muted">
         <div class="container">
-            &copy; 2023 - BucStop - <a asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a> - <a href="https://github.com/chrisseals98/BOBBY/releases/tag/v2.0.0" style="color:#C6AA76;">Version: 2.0.0</a>
+            &copy; 2023 - BucStop - <a asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a> - <a href="https://github.com/chrisseals98/BOBBY/releases/tag/v2.0.0" style="color:#C6AA76;">Version: 2.0.0</a><a> - </a><a asp-area="" asp-controller="Home" asp-action="GameCriteria">Game Criteria</a>
```

Comment:
I think a link here is good, but I think it would also be good to have a link to the page on the navbar as well. What do you think?

Reply:
I think it could be in the footer or the navbar. Probably not both though. If we think the navbar would be better I can change it to go there instead.